.github/workflows/build.yml

```diff
@@ -8,9 +8,9 @@ on:
 jobs:
     build:
         name: Build
-        runs-on: ubuntu-latest
+        runs-on: ubuntu-24.04
```

Comment:
I don't think we need this modification.
From what I understood the `ubuntu-latest` version is now mapped to 24.04. In the future will be other version, we want to stay with the latest version.

Reply:
I see your consideration. 

But the `latest` runner might bring some break changes that stop our workflow. Some actions might not be compatible with the `latest` runner, for example, <https://github.com/actions/runner-images/issues/10788>.  Using a runner with specific version is more reasonable than `latest`.

If we want to keep with the `latest` runner, we can migrate it manually to make sure all actions work very well.